images/openstack-resource-controller.yml

```diff
@@ -0,0 +1,26 @@
+content:
```

Comment:
I think the file also needs to be renamed to `ose-openstack-resource-controller.yml`, please